addons/mail/static/src/components/message/message.scss

```diff
@@ -240,8 +244,12 @@
         }
     }
 
-    &.o-has-message-selection:not(.o-selected) {
-        opacity: 0.5;
+    .not-delivred {
+        color: $red;
+    }
```

Comment:
Either missing `&`, or should be on `o_Message_sendAgain` directly?
What should be in red here?